sdk/textanalytics/azure-ai-textanalytics/src/main/java/module-info.java

```diff
@@ -6,7 +6,6 @@
 
     exports com.azure.ai.textanalytics;
     exports com.azure.ai.textanalytics.models;
-    exports com.azure.ai.textanalytics.util;
```

Comment:
We still need `util` package to be exported as this package holds the collection types that are returned to the user.